ansible/Makefile

```
@@ -268,12 +268,11 @@ nfs_cleanup: hosts
 	 rm \$$temp"
 
 destroy_ocp: hosts local-defaults.yaml
-	$(MAKE) openstack_cleanup
 	ANSIBLE_FORCE_COLOR=true ansible-playbook -i hosts \
 	-e @vars/${OSP_RELEASE}.yaml \
 	-e @local-defaults.yaml \
 	ocp_installer_destroy.yaml
-	$(MAKE) nfs_cleanup
+	$(MAKE) openstack_cleanup nfs_cleanup
```

The openstack cleanup can hang if the workers are not healthy (in `oc delete -n openstack openstacknetconfig --all` IIRC). Moving to after the ocp destroy will just cause these command to fail fast and continue the remaining cleanup tasks.